source/blog/2024-02-03-doctrine-orm-3-and-dbal-4-released.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Doctrine ORM 3 and DBAL 4 Released"
+authorName: Jonathan H. Wage
+authorEmail: jwage@doctrine-project.org
+permalink: /2024/02/03/doctrine-orm-3-and-dbal-4-released.html
+---
+
+We are thrilled to announce the release of Doctrine ORM 3.0 and DBAL 4.0.
+These releases are the culmination of over a decade of hard work across
+dozens of contributors and the Doctrine maintainers.
```

Comment:
As trivia and self-reflection (we don't have to include this in the post). Even though DBAL 3.0.0 was released in 2020, the work on DBAL 4 technically started in 2016 (https://github.com/doctrine/dbal/pull/2499, https://github.com/doctrine/dbal/commit/36522aa4562293c9df54497444992f0a89f1d0f5). So some of the improvements there are also almost 8 years old.